flipper-plugin-realm/src/components/Mermaid.tsx

```diff
@@ -50,14 +50,14 @@ mermaid.initialize({
       stroke: #f8f8f2;
       stroke-width: 1;
     }`,
-  fontFamily: 'Fira Code',
 });
 
-export default class Mermaid extends React.Component {
-  componentDidMount() {
-    mermaid.contentLoaded();
-  }
-  render() {
-    return <div className="mermaid">{this.props.chart}</div>;
-  }
+interface MermaidProps {
+  chart: string;
+}
+
+export default function Mermaid({chart}:MermaidProps) {
+  useEffect(mermaid.contentLoaded);
```

Comment:
Might not be a big deal, but this `useEffect` will be invoked on every re-render.  What is the intended behavior?

Reply:
I was just looking to port `componentDidMount()` functionality. I am not too familiar with Mermaid so was just kind of guessing but all this is meant to do is to draw the chart of schemas, assuming schemas don't change this should be barely if ever be changed, should I make the useEffect depend on the chart string?

Reply:
That may be the correct thing to do.  I can't find any documentation on that method, but I read in an SO thread that it should be called when the state is updated.  It's probably not a big deal to leave it as it is, I just wanted to ensure this was intentional